docs/getting-started/install-kura.md

```diff
@@ -63,10 +69,12 @@ For example, if the system contains the following interfaces: `wlp2s0`, `wlp3s0`
 
 ### Initial firewall configuration
 
-The initial firewall configuration will be as shown in the screenshot below.
+As done for the initial network configuration, the initial firewall setup is adjusted based on the network interface detected on the system. In the case of multiple ethernet and wireless interfaces, the configuration will be as shown in the screenshot below.
```

Comment:
I would rephrase as follows

```suggestion
Similarly to the initial network configuration, the initial firewall setup is adapted based on the network interface detected on the system. In case of multiple ethernet and wireless interfaces, the configuration will be as shown in the screenshot below.
```